core/src/main/scala/kafka/admin/AclCommand.scala

```diff
@@ -284,23 +290,21 @@ object AclCommand {
       .defaultsTo(All.name)
       .withValuesSeparatedBy(Delimiter)
 
-    val allowPrincipalsOpt = parser.accepts("allow-principals", "Comma separated list of principals where principal is in principalType:name format." +
+    val allowPrincipalsOpt = parser.accepts("allow-principal", "principal is in principalType:name format." +
       " User:* is the wild card indicating all users.")
       .withRequiredArg
       .describedAs("allow-principals")
```

Comment:
allow-principals should be allow-principal. Ditto for the description of "deny-principals".